inventory/group_vars/all.yml

```diff
@@ -10,6 +10,10 @@ local_release_dir: "/tmp/releases"
 # Random shifts for retrying failed ops like pushing/downloading
 retry_stagger: 5
 
+# Controll delayed startup/restart for services, in seconds
```

Comment:
Still not a good option. All roles depend on kubernetes/preinstall. You can put these vars there. All vars consumed inside a playbook should have a default set somewhere.